srv/modules/runners/populate.py

```diff
@@ -905,6 +921,25 @@ def engulf_existing_cluster(**kwargs):
             # Needs a storage profile assigned (which may be different
             # than the proposals deepsea has come up with, depending on
             # how things were deployed)
+            ceph_disks = local.cmd(minion, "cephinspector.get_ceph_disks_yml")
+            if not ceph_disks:
+                log.error("Failed to get list of Ceph OSD disks.")
+                return [ False ]
+
+            for minion, store in ceph_disks.items():
```

Comment:
@theanalyst - Hmmm... gives me `ValueError: too many values to unpack`
So I'd need to grab my key via `for minion in ceph_disks:` and access `ceph_disks[minion]`.  Will just keep `.items()`.